config/indexer_settings_dev.yml

```diff
@@ -2,6 +2,8 @@
 
 processing_thread_pool: 5
 hathi_overlap_path: ignorethis_hathi/
-hathi_overlap_file: final_hathi_overlap.csv
+hathi_mono_overlap_file: final_hathi_mono_overlap.csv
+hathi_multi_overlap_file: final_hathi_multi_overlap.csv
 hathi_load_period: 20200501
-overlap_file: overlap_20200518_psu.tsv
\ No newline at end of file
+overlap_file: overlap_20200518_psu.tsv
+hathi_etas: true
```

Comment:
For indexer environment level configs I made a PR recently to kinda pare them down a lot. My main thinking is that it's easier to just manage config that is the same everywhere right in the `settings` block, and that's what I'm thinking about `hathi_etas` too.